nginx/django_nginx.conf.template

```diff
@@ -0,0 +1,12 @@
+server {
+   listen       8001;
+   server_name  django_server_nginx;
+
+   location / {
+       proxy_pass ${DJANGO_APP_URL};
```

Comment:
why not through VGS?

Reply:
I used VGS proxy for requests with PII data